src/elements/Accordion/accordion.less

```diff
@@ -42,14 +41,31 @@
 			font-size: 1em;
 			text-align: center;
 			font-weight: 700;
+			transition: transform 0.5s ease;
 
 			&.arrow-down {
-				transform: rotate(180deg);
+				transform: rotate(-180deg);
+				transition: transform 0.5s ease;
 			}
 		}
 	}
 
 	.accordion-content {
-		margin-top: @ui-x-small;
+		margin: 0;
+		padding: 0;
+		opacity: 0;
+		font-size: 0;
+		transition: opacity 0.5s,
+                font-size 0.5s,
+                margin 0.5s;
+
+		&.visible {
```

Comment:
This seems like a neat way to scale the box, but there's an easier way to acheive this effect without scaling the font by using a `max-height: 0` and `overflow: hidden`, then transitioning the `max-height` to something that would likely never occur for an accordion component (like 1000px or so)